### PR TITLE
[BEAM-675] Introduce message mapper in JmsIO

### PR DESCRIPTION
**Pull request by mottyg1**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by mottyg1** [comment]
Hi @jbonofre, can you please take a look?

It's my first pull request ever, hope I did it right :)

File: sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.apache.beam.sdk.io.jms;
```
Comment (jbonofre) [inline sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java]: You need to add the Apache license header here.

File: sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ */
+package org.apache.beam.sdk.io.jms;
+
+import javax.jms.BytesMessage;
+import javax.jms.Message;
+
+/**
+ * A test class that maps a {@link javax.jms.BytesMessage} into a {@link String}.
+ */
+public class BytesMessageToStringMessageMapper implements JmsIO.MessageMapper<String> {
```
Comment (jbonofre) [inline sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java]: As this class is only used in the test, I think it's better to be an inner class of `JmsIOTest`.

Comment (jbonofre) [reply sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java]: Thoughts ?

**Comment by jbonofre** [comment]
retest this please

**Comment by jbonofre** [comment]
retest this please

File: sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java (hunk 2 above)
Comment (mottyg1) [reply sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/BytesMessageToStringMessageMapper.java]: Sounds good :)

**Pull request merged**